lms/templates/emails/password_reset_subject.txt

```diff
@@ -0,0 +1,4 @@
+{% load i18n %}
```

Comment:
I see, but all of .txt email templates are used inside this path at edX, so no need to maintain the ```babel_mako.cfg``` by adding one line just for this.